src/test/java/uk/gov/companieshouse/api/accounts/controller/DebtorsControllerTest.java

```diff
@@ -0,0 +1,123 @@
+package uk.gov.companieshouse.api.accounts.controller;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import uk.gov.companieshouse.api.accounts.AttributeName;
+import uk.gov.companieshouse.api.accounts.exception.DataException;
+import uk.gov.companieshouse.api.accounts.model.rest.notes.Debtors.Debtors;
+import uk.gov.companieshouse.api.accounts.model.validation.Errors;
+import uk.gov.companieshouse.api.accounts.service.impl.DebtorsService;
+import uk.gov.companieshouse.api.accounts.service.response.ResponseObject;
+import uk.gov.companieshouse.api.accounts.service.response.ResponseStatus;
+import uk.gov.companieshouse.api.accounts.transaction.Transaction;
+import uk.gov.companieshouse.api.accounts.utility.ApiResponseMapper;
+import uk.gov.companieshouse.api.accounts.utility.ErrorMapper;
+
+import javax.servlet.http.HttpServletRequest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class DebtorsControllerTest {
+
+    @Mock
+    BindingResult bindingResult;
+
+    @Mock
+    Debtors debtors;
+
+    @Mock
+    Transaction transaction;
+
+    @Mock
+    private HttpServletRequest request;
+
+    @Mock
+    private ApiResponseMapper apiResponseMapper;
+
+    @Mock
+    DebtorsService debtorsService;
+
+    @Mock
+    private ErrorMapper errorMapper;
+
+    @InjectMocks
+    DebtorsController controller;
+
+    private static final String COMPANY_ACCOUNTS_ID = "companyAccountsId";
+
+    @Test
+    @DisplayName("SUCCESS- Debtors resource created")
+    void createDebtorsResource() throws DataException {
+
+        when(bindingResult.hasErrors()).thenReturn(false);
+
+        when(request.getAttribute(AttributeName.TRANSACTION.getValue())).thenReturn(transaction);
+
+        ResponseObject responseObject = new ResponseObject(ResponseStatus.CREATED,
+                debtors);
+        when(debtorsService.create(debtors, transaction, COMPANY_ACCOUNTS_ID, request))
+                .thenReturn(responseObject);
```

Comment:
Could we format the file and the other files? so that the indentation matches the coding standards: 

https://github.com/companieshouse/styleguides/blob/master/java.md#indentation